reactioncommerce/reaction
# Fix on Localized elements not updating on currency/language change

**Resolves #2593**

TEST BY:

- Cloning this branch
-  Setup `reaction.json` to have the open exchange rates API key or use the [standard `reaction.json`](https://github.com/reactioncommerce/rc-config-files/blob/master/reaction.json)
- Start reaction and login as admin
- Change currency or i18n in a page where there are multiple localized elements and notice that components update on currency/language change
- Test this also as a guest/non-admin user
